client/coniksclient/internal/cmd/common.go

```diff
@@ -39,5 +40,5 @@ func loadConfigOrExit(cmd *cobra.Command) *client.Config {
 
 // append "\r\n" to msg and then write to terminal in raw mode.
 func writeLineInRawMode(term *terminal.Terminal, msg string) {
-	term.Write([]byte(msg + "\r\n"))
+	term.Write([]byte("<" + time.Now().Format("15:04:05.999999999") + "> " + msg + "\r\n"))
```

Comment:
Example output:
`<15:43:01.090076899> [+] Are you trying to update your binding? Unfortunately, KeyChange isn't supported yet.`
One question to the others: Should we make this optional? 
Otherwise I'm fine with this pull!